products/sle/main.pm

```diff
@@ -1215,8 +1204,20 @@ elsif (get_var("Y2UITEST")) {
     # setup $serialdev permission and so on
```

Comment:
now as you split it the same way as used for o3 you would need to also sync the test suites, i.e. define a yast ui test suite.